operator-framework-core/src/main/java/io/javaoperatorsdk/operator/LeaderElectionManager.java

```diff
@@ -67,7 +67,7 @@ private void init(LeaderElectionConfiguration config) {
                 config.getRenewDeadline(),
                 config.getRetryPeriod(),
                 leaderCallbacks(),
-                true,
+                false,
```

Comment:
Can you add a comment to explain why setting this to true prevented stop from being called, please?

Reply:
added